packages/components-native/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@jobber/components-native",
+  "version": "0.0.0",
+  "license": "MIT",
+  "main": "dist/esm/src/index.js",
+  "module": "dist/esm/src/index.js",
+  "types": "dist/types/src/index.d.ts",
+  "files": [
+    "dist",
+    "src",
+    "!**/__tests__",
+    "!**/__fixtures__",
+    "!**/__mocks__"
+  ],
+  "scripts": {
+    "clean": "rm -rf dist/* tsconfig.tsbuildinfo",
+    "build": "npm run build:clean && npm run compile",
+    "bootstrap": "npm run build",
+    "prepack": "npm run build",
+    "compile": "tsc -b ./tsconfig.esm.json ./tsconfig.types.json",
+    "build:clean": "rm -rf ./dist"
+  },
+  "dependencies": {
+    "@jobber/design": "*",
+    "react-native-svg": "^12.4.4",
+    "ts-xor": "^1.1.0"
+  },
+  "devDependencies": {
+    "@types/node": "*",
+    "@types/react": "*",
+    "@types/react-native": "*",
+    "metro-react-native-babel-preset": "^0.76.0",
+    "typescript": "*"
+  },
+  "peerDependencies": {
+    "@babel/core": "*",
+    "react": "*",
+    "react-native": "*"
```

Comment:
I am hesitant about an unpinned dependency here. If when packaging it grabs from the main package.json this is fine but if not we should probably have a version specified

Reply:
Also applies to the react version

Reply:
Do you think we should put the same version as we have in jobber-mobile?

Reply:
I think so. We may want a couple patch versions below but I think matching jobber-mobile for now is good. We don't really want to have things be broken on other versions

Reply:
I got an peer dependency issue because `react-native@0.69.10` (version installed on jobber-mobile), has a peer dependency on `react@18.0.0` and we currently have `react@18.2` installed.

I think it's safe for us to use the latest version in `components-native`. I added a minimum peer dependency version to be the same one as jobber-mobile